### DISCUSSION TITLE
Thank you - it is working! :-)

### DISCUSSION THREAD

**Discussion by dsteinkopf** [jomjol/AI-on-the-edge-device#1279] [Show and tell]
I just would like to say thank you and tell you some of my experiences:

I am using this device: https://www.az-delivery.de/products/esp32-cam-modul-esp32-wifi-bluetooth-modul-inklusive-kamera?variant=32376084136032

Power supply seems really crucial (as states several times in the project). I had to try several supplies until I found a working one. I also can confirm that even a long cable may be a problem. I am now working with a 5V/4A supply using not too long cables.

**Comment by dsteinkopf** [comment]
BTW. After I had understood how to use the web interface for defining the ROI (which took quite a while) - I was really surprised about the recignition quality.

Good stuff! :-)

**Comment by jomjol** [comment]
Thank for your feedback. I think the power supply also strongly depends on the quality of the esp32cam. you never know, which stuff you really get from china :-).
I have some working with 3 devices on a 1.5A power supply, and one with a 5m long telefon cable. But I also had the experience in another case, that I needed to test several power supplies, before one was working.

**Comment by dsteinkopf** [comment]
BTW. It seems that digit (ROI) names must not contain comma (","). This should be mentioned or prevented.
I first tried to name the fraction digits sth. like "0,1", "0,01", ... which did not work. After deleting the comma, it worked.

**Comment by jomjol** [comment]
good point! I make an issue out of that.
